src/components/CertificateViewer.js

```diff
@@ -25,19 +24,13 @@ const renderVerifyBlock = props => (
   />
 );
 
-const renderAssetInfo = props => {
-  const { tokenRegistry, tokenId } = getAssetInfo(props.document);
-
-  return tokenRegistry ? <AssetInfo registryAddress={tokenRegistry} tokenId={tokenId} /> : undefined;
-};
-
 const renderHeaderBlock = props => {
   const renderedVerifyBlock = renderVerifyBlock(props);
   return (
     <div className={`container-fluid ${styles["pd-0"]} ${styles.container}`}>
       <div className="row">
         <div>{renderedVerifyBlock}</div>
-        <div>{renderAssetInfo(props)}</div>
+        <AssetInfo document={props.document} />
```

Comment:
Comment above to move this out of here. Can rename AssetInfo to something like `ManageAssetToggle`